libfreerdp/core/test/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ if (BUILD_FUZZERS)
 		target_link_libraries(${TestName} freerdp winpr freerdp-client fuzzer_config)
 		add_test(${TestName} ${TESTING_OUTPUT_DIRECTORY}/${MODULE_NAME} ${TestName})
 		set_target_properties(${TestName} PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${TESTING_OUTPUT_DIRECTORY}")
+		set_target_properties(${TestName} PROPERTIES POSITION_INDEPENDENT_CODE ${BUILD_SHARED_LIBS})
```

Comment:
why do you need to disable `POSITION_INDEPENDENT_CODE` ? if so, do you want it globally disabled or really only for this target?
might be a good idea to change the setting in the main `CMakeLists.txt` to an `option` so you can override it from outside

Reply:
> why do you need to disable POSITION_INDEPENDENT_CODE ? if so, do you want it globally disabled or really only for this target?

I thought it is required for building static binary, but after double-checking it is not so.

Removed this hunk. If I'll need it, there is a `CMAKE_POSITION_INDEPENDENT_CODE` that can be disabled from outside.